src/styles/core/typography/_font-styles.scss

```diff
@@ -194,5 +194,8 @@ $font-styles: (
   ),
   nav-menu-collapsible: (
     large: (nav_14_14, nav)
+  ),
+  small-text: (
```

Comment:
I think we could probably just use `body-reduced-tight` instead of adding a new duplicate style, right?

Reply:
yep, done.